docs/guides/designing-the-delta-format.md

```diff
@@ -258,7 +258,7 @@ var delta = [{
 
 #### Pitfalls
 
-First, we should be clear that this index must refer to the index in the document **before** any Operations are applied. Otherwise, a later Operation may delete a previous insert, unformat a previous format, etc, which would violate compactness.
+First, we should be clear that an index must refer to its value in the document **before** any Operations are applied. Otherwise, a later Operation may delete a previous insert, unformat a previous format, etc., which would violate compactness.
```

Comment:
I'd prefer "its position" over "its value" since it is technically more correct. I do agree on using a different word than index though.